Assets/Scripts/Game/GameManager.cs

```diff
@@ -47,16 +47,19 @@ private void Awake()
     [SerializeField] private float maxBaseSpeed = 4; // The maximum speed of the game independent of the catchup mechanic
     [SerializeField] private float accelerationCurve = 1.5f; // The speed at which the "camera" catches up to player (lower is faster)
     [SerializeField] private float decelerationCurve = 0.5f; // The speed at which the "camera" slows back down (lower is faster)
-    
+
     private float baseSpeed; // The speed of the game independent of the catchup mechanic
     private float initSpeed = 1; // The initial speed  
     private float decelerationTolerance = 0.1f; // The tolerance for the speed to be considered back to normal
     private float currentVelocity = 0f;
-  
+
 
     private bool isCatchingUp;
     public bool playerInCatchupZone = false;
 
+    [Header("Crow settings")]
+    [SerializeField] private GameObject crowManagerParent;
+    private CrowManager crowManager;
```

Comment:
Why not expose the CrowManager component itself in the inspector? Then you don't need to get the component from the GameObject on lines 157 and 167